office_space_allocation/main.py

```diff
@@ -0,0 +1,310 @@
+import os
+import re
+import sqlite3
+import subprocess
+import sys
+from colorama import Fore, Style
+
+from tabulate import tabulate
+
+from office_space_allocation import amity, office, staff, livingspace, fellow, utilities, db
+
+main_amity = None
+
+
+def init_amity():
+    """
+    Initialises the Amity class
+    """
+    global main_amity
```

Comment:
I do not think line 12 is required since on line 19 you are redefining main_amity as a global variable.